api/apirouter.go

```diff
@@ -140,7 +140,8 @@ func NewAPIRouter(app *appContext, userRealIP bool) apiMux {
 	mux.Route("/address", func(r chi.Router) {
 		r.Route("/{address}", func(rd chi.Router) {
 			rd.Use(m.AddressPathCtx)
-			rd.Use(m.DenySpecificAddr)
+			rd.Use(app.ZeroAddrDenier)
```

Comment:
`app.ZeroAddrDenier(params)`

Reply:
according docs https://godoc.org/github.com/go-chi/chi#Mux.Use
Use accepts handler without parameters and returns handler without parameters.

Reply:
Please see the suggested changes.  `m.ZeroAddrDenier(params)` would return such a function:

    func ZeroAddrDenier(params *chaincfg.Params) func(http.Handler) http.Handler {

Reply:
where you get params from?